ironic/conf/drac.py

```diff
@@ -21,7 +21,13 @@
                min=1,
                help=_('Interval (in seconds) between periodic RAID job status '
                       'checks to determine whether the asynchronous RAID '
-                      'configuration was successfully finished or not.'))
+                      'configuration was successfully finished or not.')),
```

Comment:
Hey Pradip, I don't think we want to be able to set the priority of these cleaning steps in ironic.conf for now.  Can you undo this change?

Reply:
Done